### PR TITLE
Make GetLoadedProfiles reusable for apparmor

### PR DESCRIPTION
**Pull request by gavinfish**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

`GetLoadedProfiles` is used to get profiles for AppArmor which is not narrowed in validator. Move it to other profile related functions like `SetProfileName` in helper.go. This refactor could help to reduce duplicated code in k/k.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
none
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: test/images/apparmor-loader/loader.go [hunk 1]
```diff
@@ -17,19 +17,18 @@ limitations under the License.
...
 	"io/ioutil"
 	"os"
 	"os/exec"
-	"path"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/security/apparmor"
```
Comment (liggitt) [inline test/images/apparmor-loader/loader.go:31]: I thought dependencies from test images into k8s.io/kubernetes were not desired?

**Comment by dims** [comment]
/approve cancel

hmm i missed that!

File: test/images/apparmor-loader/loader.go (hunk 1 above)
Comment (gavinfish) [reply test/images/apparmor-loader/loader.go:31]: @liggitt Thanks for sharing that. If so, can we just revert the change and clean the TODO?

```
// TODO: This is copied from k8s.io/kubernetes/pkg/security/apparmor.getLoadedProfiles.
//       Refactor that method to expose it in a reusable way, and delete this version.
```

**Pull request closed without merging**